components/fastly_frontend/main.tf

```diff
@@ -134,6 +134,24 @@ resource "fastly_service_v1" "frontend" {
     }
   }
 
+  # Override: Remove 'X-Frame-Options' on URL(s) that we do
+  # want to allow embedding from external domains:
+  condition {
+    type      = "RESPONSE"
+    name      = "allow-embed"
+    statement = "req.url ~ \"^/us/about/voter-registration-thank-you\""
+  }
+
+  header {
+    name               = "Allow Embed on Path(s)"
+    type               = "response"
+    action             = "delete"
+    destination        = "http.X-Frame-Options"
+    response_condition = "allow-embed"
```

Comment:
This `response_condition` is what ties the two together? Not super familiar with the syntax of this

Reply:
Yep, exactly! By default, changes like this are applied to all requests. The `condition` block allows us to segment something on a subset of requests/responses, and then we can attach that to other behavior.

We also use this to [route traffic to different backends](https://github.com/DoSomething/infrastructure/blob/daec9a076a6f4f5370f0d4d2adc182ac753c50f2/components/fastly_backends/main.tf#L79-L91) based on `req.host` in our backend property.

Reply:
Ah those examples are helpful, thanks!